lib/mix/tasks/nerves/new.ex

```diff
@@ -152,6 +154,9 @@ defmodule Mix.Tasks.Nerves.New do
     end
   end
 
+  defp run(app, _mod, _path, _opts) when app in @reserved_names,
+    do: Mix.raise("New projects cannot be named '#{app}'")
```

Comment:
I think you should use `@reserved_names` instead of `app` here, wdyt?

Reply:
The reason I decided to use app is to not advertise the reserved names. Either because it would not be obvious why they are reserved or of the list of names gets too long.
I appreciate you could potentially try 'nerves' and then correct the error but chose another reserved name and might be frustrated, but that seems very unlikely (also impossienwith just one name in the list.)

Reply:
Agreed, make sense to not expose the whole list of reserved names, it could be confusing for the user. Thanks for the sharing your thoughts.

Reply:
my pleasure - this is the same thing we do for Hex BTW